src/theme/colors.js

```diff
@@ -1,4 +1,65 @@
 const colors = {
+  tmBrand: "#026cdf",
```

Comment:
I think we should keep this file the same and do the mapping in `colorThemes.js`